### PR TITLE
Draft feature part blog

### PR DESCRIPTION
**Pull request by andyfengHKU**

### CODE REVIEW

File: src/content/post/2024-07-31-kuzu-v-0.5.0.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ heroImage: "/img/default.png"
 categories: ["release"]
 authors: ["team"]
 tags: ["cypher", "extensions"]
-draft: true
+draft: false
```
Comment (prrao87) [inline src/content/post/2024-07-31-kuzu-v-0.5.0.md:10]:
```suggestion
draft: true
```

File: src/content/post/2024-07-31-kuzu-v-0.5.0.md [hunk 2]
```diff
@@ -96,7 +95,7 @@ LOAD FROM uw.person RETURN *
 ```
 
 ### Json
-This release adds Json support through extension.
+This release adds Json support through extension. User can now directly scan and ingest data through a json file. It is also possible to store json natively as data type in Kùzu.
```
Comment (prrao87) [inline src/content/post/2024-07-31-kuzu-v-0.5.0.md:98]:
```suggestion
This release adds JSON support through extension. User can now directly scan and ingest data through a json file. It is also possible to store json natively as data type in Kùzu.
```

File: src/content/post/2024-07-31-kuzu-v-0.5.0.md [hunk 3]
```diff
@@ -136,22 +135,74 @@ CREATE NODE TABLE Person (id BIGINT, name STRING, age BIGINT, PRIMARY KEY(id));
...
 
 ## Performance improvements
 
 ### MVCC
 
 Version 0.5.0 of Kùzu brings numerous updates and improvements to the multi-version concurrency control (MVCC) protocol.
 
-#### INT128 compression
+### Hint join order
+Picking an optimal join order can be challenging when it comes to a large query graph. We introduce join order hints to by-pass optimizer and enforce a specific join order. E.g.
```
Comment (prrao87) [inline src/content/post/2024-07-31-kuzu-v-0.5.0.md:187]: @andyfengHKU could you add a sentence after the code snippet below explaining the benefit? What performance improvements did we notice? etc.

File: src/content/post/2024-07-31-kuzu-v-0.5.0.md [hunk 4]
```diff
@@ -136,22 +135,74 @@ CREATE NODE TABLE Person (id BIGINT, name STRING, age BIGINT, PRIMARY KEY(id));
...
+```
+MATCH (a:person)<-[e1:knows]-(b:person)-[e2:knows]->(c:person), (a)-[e3:knows]->(c)
+HINT (((a JOIN e1) JOIN b) MULTI_JOIN e2 MULTI_JOIN e3) JOIN c
+RETURN COUNT(*)
+```
+
 
-### Hash index
+### INT128 compression
+We extended our compression algorithm to further compress `INT128`. This compression should works for dataset with `INT128` or `UUID` column. The `INT128` compression algorithm is highly inspired by DuckDB.
```
Comment (prrao87) [inline src/content/post/2024-07-31-kuzu-v-0.5.0.md:205]:
```suggestion
We extended our compression algorithm to further compress `INT128`. This compression should work for dataset with `INT128` or `UUID` column. The `INT128` compression algorithm is highly inspired by the one used by DuckDB. 
```

**Review by prrao87 — CHANGES_REQUESTED** [review]

File: src/content/post/2024-07-31-kuzu-v-0.5.0.md (hunk 3 above)
Comment (andyfengHKU) [reply src/content/post/2024-07-31-kuzu-v-0.5.0.md]: Done. Also moved this to feature section instead of performance improvement section.

**Review by prrao87 — APPROVED** [review]

**Pull request merged**